ui/app/common/displaycontrols/observation/directives/bahmniObservation.js

```diff
@@ -50,7 +50,7 @@ angular.module('bahmni.common.displaycontrol.observation')
                     });
 
                     if (formObservations.length > 0) {
-                        formRecordTreeBuildService.build($scope.bahmniObservations, $scope.hasNoHierarchy);
+                        formRecordTreeBuildService.build($scope.bahmniObservations);
```

Comment:
@rohit-yawalkar build() expecting 2 parameters. can you send "false" instead of completly removing the parameter?